python/TestHarness/util.py

```diff
@@ -96,17 +98,36 @@ def runCommand(cmd, cwd=None):
         output = 'ERROR: ' + output
     return output
 
+# Execute a command and return the process, temporary output_file and time the process was launched
+def returnCommand(tester, command):
```

Comment:
Weird name: returnCommand()? Why not launchProcess(). This is really close to "getCommand()"